docs/proposals/object-search.md

```diff
@@ -0,0 +1,160 @@
+- [Object Search Service](#object-search-service)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+  - [Proposal](#proposal)
+    - [Use Cases](#use-cases)
+  - [Design Details](#design-details)
+    - [CRD API Group and Version](#crd-api-group-and-version)
+    - [Object search service CRD](#object-search-service-crd)
+    - [Object search service type definition](#object-search-service-type-definition)
+      - [Validation](#validation)
+    - [Object search service sample](#object-search-service-sample)
+  - [Controller Design](#controller-design)
+    - [Object search service Controller](#object-search-service-controller)
+    - [Downstream Controller](#downstream-controller)
+    - [Upstream Controller](#upstream-controller)
+    - [Details of api between GM(cloud) and LC(edge)](#details-of-api-between-gmcloud-and-lcedge)
+    - [Flow of object search service creation](#flow-of-object-search-service-creation)
+  - [Workers Communication](#workers-communication)
+# Object Search Service
+## Motivation
+Object search is an important technology in the field of computer vision, which is widely used in security monitoring, intelligent transportation, etc. Generally, online object search applications have stringent latency constraints, which cannot be met by cloud computing schemes. Object search schemes based on edge computing have the characteristics of low latency and data privacy security, and are the mainstream technology trend. However, the amount of feature matching computation increases exponentially with the expansion of the search object scale, and a single edge computing node is difficult to support large-scale object search. Multi-edge collaborative computing can accelerate large-scale object search applications and improve the search accuracy, which is the future development trend. We propose the first open source end-to-end multi-edge collaborative object search solution. Based on KubeEdge's cloud-edge collaboration and resource management capabilities, we utilize multiple edge computing nodes to execute the AI inference tasks of object search in parallel. Our solution can not only reduce delay and improve throughput, but also will bring accuracy promotion. In addition, our solution will also support efficient offline object search.
```

Comment:
can you split into multi segments? just one long segment is hard to read

Reply:
OK, I've divided it into three paragraphs.